RELEASE.md

```diff
@@ -46,6 +46,7 @@
 ### Fix
 
 - La variazione "Allegati" del blocco Elenco mostra ora la vista File quando un utente è autenticato, e presenta il comportamento richiesto (mostra anteprima PDF se l'opzione è selezionata e il file è un PDF, scarica il file se il file non è un PDF o l'opzione non è selezionata) se l'utente è anonimo.
+- Lo sfondo del blocco elenco viene ora visualizzato nella variante colore corretta sia in modalità visualizzazione che in modalità modifica.
```

Comment:
Mergiando l'ultimo main si è mescolata male questa modifica, è finita in una release già fatta, dovresti per favore spostarla in una nuova release